spec/acceptance/bird_spec.rb

```diff
@@ -4,14 +4,28 @@
   context 'default parameters' do
     # Using puppet_apply as a helper
     it 'works idempotently with no errors' do
-      pp = <<-EOS
-      class { '::bird':
-        manage_service => true,
-        service_v4_enable => true,
-        enable_v6 => true,
-        service_v6_enable => true
-      }
-      EOS
+      # The RedHat package only provides an example configuration for IPv4
+      # Therefor we can't start the daemon for IPv6
+      pp = if fact('os.family') == 'RedHat'
```

Comment:
This is complex to read. Should we simplify it a bit?

Reply:
this is how I had it initially:
```puppet
      if fact('os.family') == 'RedHat'
        pp =  <<-EOS
        class { 'bird':
          manage_service => true,
          service_v4_enable => true,
          enable_v6 => true,
          service_v6_enable => true,
          service_v6_ensure => 'stopped',
        }
        EOS
      else
        pp = <<-EOS
        class { 'bird':
          manage_service => true,
          service_v4_enable => true,
          enable_v6 => true,
          service_v6_enable => true,
        }
        EOS
      end
```

but rubocop doesn't like it (Style/ConditionalAssignment) and updated it to what's currently in the PR.

Reply:
I find this VASTLY more readable and Rubocop is wrong. I know you're surprised.